app/models/event.py

```diff
@@ -229,6 +231,39 @@ def get_staff_roles(self):
         """returns only roles which are staff i.e. not attendee"""
         return [role for role in self.roles if role.role.name != ATTENDEE]
 
+    def get_tz_aware_time(self, time):
+        return pytz.timezone(self.timezone).localize(time)
+
+    @hybrid_property
+    def start_time_tz(self):
+        if not self.start_time:
+            return None
+        return self.get_tz_aware_time(self.start_time)
+
+    @hybrid_property
+    def end_time_tz(self):
+        if not self.end_time:
+            return None
+        return self.get_tz_aware_time(self.end_time)
+
+    @hybrid_property
+    def schedule_published_on_tz(self):
+        if not self.schedule_published_on:
+            return None
+        return self.get_tz_aware_time(self.schedule_published_on)
+
+    @start_time_tz.setter
```

Comment:
@niranjan94 I have declared `setter` for hybrid properties. Shouldn't it manage POST requests. I am getting error in tests - 
```
File "/home/travis/build/fossasia/open-event-orga-server/app/api/helpers/helpers.py", line 235, in create_model

    new_model = model(**data)

TypeError: __init__() got an unexpected keyword argument 'start_time_tz'
```